src/exo2 - Combinators/exo2.ts

```diff
@@ -168,11 +228,39 @@ export const shootOption: (character: Character) => Option<Damage> =
 // perform mapping and filtering at the same time by applying a function
 // of type `A => Option<B>` over the collection.
 
+
+
+
+//****** I looked at the solution to get help on this one. **********//
+
 export interface TotalDamage {
   [Damage.Physical]: number;
   [Damage.Magical]: number;
   [Damage.Ranged]: number;
 }
+/*
+Why doesn't the code in the line below work as a replacement for option 1 on line 252?
+    readonlyArray.filterMap(flow(option.filter(isWizard), option.map(character => character))),
```

Comment:
Let's call about this one, as I'm not sure I get what is the question :)

Reply:
We can skip this one as I can't figure out what I meant here either. :/